packages/core/src/file-context.ts

```diff
@@ -56,10 +56,9 @@ export class FileContext {
 
   get ignoredChecksByLine(): Map<number, string[]> {
     if (this._parsedIgnoreChecksByLine) return this._ignoredChecksByLine;
-
-    this.source.forEachChild(n => {
-      const lineNumber = this.source.getLineAndCharacterOfPosition(n.pos).line;
-      const trimmedNodeText = n.getFullText().trim();
+    let lineNumber = 0;
+    this.content.split(/\r?\n/).forEach(n => {
```

Comment:
Nit - `forEach` will provide the line number as the second param.
```suggestion
    this.content.split(/\r?\n/).forEach((n, lineNumber) => {
```